src/convertToObject.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const stylesArray = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter(item => item !== '');
```

Comment:
I don't think you really need these 2 lines.
1. You trim it later anyways
2. Just add a condition inside `forEach` to check if both value and key are defined then ...